Lib/test/test_cmd_line.py

```diff
@@ -369,6 +371,8 @@ def test_closed_stdout(self):
     # Issue #7111: Python should work without standard streams
 
     @unittest.skipIf(os.name != 'posix', "test needs POSIX semantics")
+    @unittest.skipUnless(SUBPROCESS_SUPPORTS_PREEXEC,
+                         "test needs preexec support in subprocess.Popen")
```

Comment:
Hum sorry, I didn't notice that the constant is only used once. So I don't think that having a constant is justified. Remove it and just write:

```
    @unittest.skipIf(sys.platform == "vxworks",
                     "test needs preexec support in subprocess.Popen")
```